pkg/reconciler/autoscaling/kpa/resources/metric.go

```diff
@@ -18,14 +18,34 @@ package resources
 
 import (
 	"context"
+	"time"
 
 	"github.com/knative/serving/pkg/apis/autoscaling/v1alpha1"
 	"github.com/knative/serving/pkg/autoscaler"
 )
 
 // MakeMetric constructs a Metric resource from a PodAutoscaler
 func MakeMetric(ctx context.Context, pa *v1alpha1.PodAutoscaler, config *autoscaler.Config) *autoscaler.Metric {
+	stableWindow, ok := pa.Window()
+	if !ok {
+		stableWindow = config.StableWindow
+	}
+	// Look for a panic window percentage annotation.
+	panicWindowPercentage, ok := pa.PanicWindowPercentage()
+	if !ok {
+		// Fall back on cluster config.
+		panicWindowPercentage = config.PanicWindowPercentage
+	}
+	if panicWindowPercentage == 0.0 {
+		// Fall back on a hard coded value.
+		panicWindowPercentage = 10
```

Comment:
Should these defaults be part of the constants? (same for the MakeDecider defaults)

Reply:
Yes.

Reply:
Actually, I decided to remove these hard coded defaults because `Config` already has defaults.  So there *will* be a value.  We just happen to create the config directly in the unit tests, so a small update there makes the problem go away.